local_server/qwen3_tts_server/local_server.py

```
@@ -0,0 +1,261 @@
+            logger.info(f"🚀 语音引擎初始化完成，总耗时: {time.time() - t0:.2f}s")
+        except Exception as e:
+            logger.error(f"❌ 加载引擎异常: {e}")
+            import traceback
+            logger.error(traceback.format_exc())
+
+    async def handle_tts(self, websocket):
+        logger.info(f"客户端已连接: {websocket.remote_address}")
+        loop = asyncio.get_running_loop()
+
+        text_buffer = ""
+        session_cfg = {"voice": None, "sample_rate": 24000}
```

Ruff (0.14.14)

[error] Local variable `session_cfg` is assigned to but never used

Remove assignment to unused variable `session_cfg`

(F841)